src/client/java/teammates/client/scripts/DataMigrationForSanitizedDataInStudentProfileAttributes.java

```diff
@@ -0,0 +1,107 @@
+package teammates.client.scripts;
+
+import teammates.client.remoteapi.RemoteApiClient;
+import teammates.client.scripts.util.DataMigrationForSanitizedDataHelper;
+import teammates.client.scripts.util.LoopHelper;
+import teammates.common.exception.EntityDoesNotExistException;
+import teammates.common.exception.InvalidParametersException;
+import teammates.logic.core.ProfilesLogic;
+import teammates.common.datatransfer.attributes.StudentProfileAttributes;
+import teammates.storage.api.ProfilesDb;
+
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * Script to desanitize content of {@link StudentProfileAttributes} if it is sanitized.
+ * Fields {@link StudentProfileAttributes#shortName}, {@link StudentProfileAttributes#email},
+ * {@link StudentProfileAttributes#institute}, {@link StudentProfileAttributes#nationality},
+ * {@link StudentProfileAttributes#gender}, {@link StudentProfileAttributes#moreInfo}
+ * are no longer sanitized before saving and these fields are expected to be in their unsanitized form.
+ * This script desanitizes these fields of exisiting StudentProfileAttributes if they are sanitized so that
+ * all profiles will have unsanitized values in these fields.
+ */
+public class DataMigrationForSanitizedDataInStudentProfileAttributes extends RemoteApiClient {
+    private static final boolean isPreview = false;
+    private ProfilesDb profilesDb = new ProfilesDb();
```

Comment:
Most *Logic classes contain the getAll* methods, even though these methods are only used in scripts as well. Most scripts access these methods through *Logic, instead of *Db as well. 
Putting this method in ProfilesLogic may also help developers who may need the method in the future find it more easily. Interacting with logic might be a better choice, so I will remove profilesDb and update through profilesLogic instead.